src/browser/browser.zig

```diff
@@ -335,6 +335,8 @@ pub const Page = struct {
 
         // if the url is about:blank, nothing to do.
         if (std.mem.eql(u8, "about:blank", request_url.raw)) {
+            var about_blank = AboutBlank{};
```

Comment:
`AboutBlank` is pretty clever, but this would work too I think:

```zig
var fbs = std.io.fixedBufferStream("");
try self.loadHTMLDoc(fbs.reader(), "utf-8");
```